tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
```

Comment:
Shouldn't be the same config as our other libraries?

Reply:
Yeah, fixed

But should we output commonjs or esm?

Reply:
> But should we output commonjs or esm?

Isn't that the million-dollar question? Drei & r3f does `commonjs`... 

ngl, I'm yet to really learn the difference.

Reply:
Then let's stay with commonjs

Reply:
update: I just googled it, I do know the difference.

but I agree with you, stay with common